website/docs/getting-started/setup/instance-configuration/authentication/google-login.md

```diff
@@ -37,21 +37,10 @@ To enable Google Sign-in, go to the [Google API console](https://console.cloud.g
 ## Configure Appsmith instance  
 
 :::caution
-* The Client ID and Client Secret entered aren't verified by any means. Please test it out manually before disabling the Form login authentication method. If Form login is disabled before testing the Google authentication configuration setup, it may lead to the user getting stuck on the login screen if the Google configuration is invalid. You’ll have to update the environment variables manually to fix this, as you can't edit it via the Admin settings page since you won’t be logged in anymore.
-
+* Please manually test the Client ID and Client secret before disabling the Form login authentication. If the google configuration is invalid, the user is stuck on the login screen. To fix this, you’ll have to update the environment variables manually.
 * If you are running appsmith behind an ELB / Proxy, [configure Nginx](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
 :::
-
-There are two ways to configure the self-hosted Appsmith instance - Admin Settings and Environment Variables:
-
-**On Admin Settings screen (recommended):**  On your self-hosted instance, navigate to the [Admin Settings page](/getting-started/setup/instance-configuration/). 
-
-* Under the **Authentication** tab,  provide the Client ID and Client Secret generated in the previous steps.
-* Once you've entered your Client ID and Client Secret, click the **Save & Restart** button at the bottom of the page. This prompts your Appsmith instance with the new configuration in effect.
-
-![](/img/as_google_auth_config.png)
-
-**Using environment variables:** Update the values for the following keys in your configuration file.
```

Comment:
Let's add a short intro for Configuring Appsmith Instance, which tells users that there are below ways in which the instance could be configured:
* Admin Settings(Add links to its subsection)
* Environment Variables(Add links to its subsection)